src/modules/Packages.rb

```diff
@@ -1154,9 +1156,10 @@ def ComputeSystemPackageList
 
       # bnc #431580
       # New API for packages selected by other modules
-      packages_proposal_all_packages = PackagesProposal.GetAllResolvables(
-        :package
-      )
+      # use both mandatory and optional packages
+      packages_proposal_all_packages = PackagesProposal.GetAllResolvables(:package, optional: true)
+      packages_proposal_all_packages.concat(PackagesProposal.GetAllResolvables(:package))
```

Comment:
below is log message about no required packages, but in fact it is no all packages, required and also optional.